website/pages/docs/theming/component-style.mdx

```diff
@@ -166,7 +166,7 @@ function Button(props) {
   const styles = useStyleConfig("Button", { size, variant })
 
   // 3. Pass the computed styles into the `sx` prop
```

Comment:
```suggestion
  // 3. Pass the computed styles into the `__css` prop
```
 
We also need to explain what the heck the `__css` means to avoid confusion. We need to explain that it is similar in spirit to the `sx` prop but allows for prop overrides.

User can use either `sx` or `__css` prop depending on their "strictness" needs.

Reply:
Very true - What do you think about adding a section about the `__css` prop at [The sx prop](https://chakra-ui.com/docs/features/the-sx-prop). `__css` does the same as `sx`, it only has a different priority.

Reply:
I'd prefer we mention it in the theming docs only to avoid confusion. We can keep the `sx` prop in the docs and add a section for `Props Priority/Override` in the theming props that replace the `sx` with `__css`.